src/hotspot/share/runtime/os.cpp

```diff
@@ -22,6 +22,9 @@
  *
  */
 
+// Stopgap fix until FORBID_C_FUNCTION can work properly with LTO
+#define DISABLE_POISONING_STOPGAP
```

Comment:
This prevents precompiled headers from being used for this file.  `-Winvalid-pch` will warn if enabled.